### PR TITLE
feat: add text utilities and lists

### PR DESCRIPTION
**Pull request by devdeadviz**
# Change Logs
## Screenshots
Screenshot (137)
Screenshot (136)
## Deploy preview link: 
https://deploy-preview-5--arcui.netlify.app/

### CODE REVIEW

File: styles/variable.css [hunk 1]
```diff
@@ -15,4 +15,5 @@
   --padding: 0.25rem;
   --margin: 0.25rem;
   --default-font-size: 87.5%;
+  --light-gray-color: #d3d3d3;
 }
```
Comment (haarsh24) [inline styles/variable.css:19]: Declare and use this color

**Review by haarsh24 — APPROVED** [review]

**Review by haarsh24 — APPROVED** [review]
Looks good , you can merge.

**Review by Rishi05051997 — COMMENTED** [review]
Good to go

File: styles/variable.css (hunk 1 above)
Comment (devdeadviz) [reply styles/variable.css:19]: Following the same approach, will resolve it at end.

**Pull request merged**